--- codeforfukui
+++ wikidata
@@ -12,6 +12,7 @@
 函館市,www.city.hakodate.hokkaido.jp
 小樽市,www.city.otaru.lg.jp
 旭川市,www.city.asahikawa.hokkaido.jp
+室蘭市,www.city.muroran.lg.jp
 室蘭市,www.city.muroran.lg.jp
 釧路市,www.city.kushiro.lg.jp
 帯広市,www.city.obihiro.hokkaido.jp
@@ -66,7 +67,7 @@
 黒松内町,www.kuromatsunai.com
 蘭越町,www.town.rankoshi.hokkaido.jp
 ニセコ町,www.town.niseko.lg.jp
-真狩村,makkari.info
+真狩村,www.vill.makkari.lg.jp
 留寿都村,www.vill.rusutsu.lg.jp
 喜茂別町,www.town.kimobetsu.hokkaido.jp
 京極町,www.town-kyogoku.jp
@@ -123,7 +124,7 @@
 猿払村,www.vill.sarufutsu.hokkaido.jp
 浜頓別町,www.town.hamatonbetsu.hokkaido.jp
 中頓別町,www.town.nakatombetsu.hokkaido.jp
-枝幸町,www.town.esashi.hokkaido.jp
+枝幸町,www.esashi.jp
 豊富町,www.town.toyotomi.hokkaido.jp
 礼文町,www.town.rebun.hokkaido.jp
 利尻町,www.town.rishiri.hokkaido.jp
@@ -141,7 +142,7 @@
 湧別町,www.town.yubetsu.lg.jp
 滝上町,www.town.takinoue.hokkaido.jp
 興部町,www.town.okoppe.lg.jp
-西興部村,www.vill.nishiokoppe.lg.jp
+西興部村,www.vill.nishiokoppe.hokkaido.jp
 雄武町,www.town.oumu.hokkaido.jp
 大空町,www.town.ozora.hokkaido.jp
 豊浦町,www.town.toyoura.hokkaido.jp
@@ -221,7 +222,7 @@
 東通村,www.vill.higashidoori.lg.jp
 風間浦村,www.kazamaura.jp
 佐井村,www.vill.sai.lg.jp
-三戸町,town.sannohe.aomori.jp
+三戸町,www.town.sannohe.aomori.jp
 五戸町,www.town.gonohe.aomori.jp
 田子町,www.town.takko.lg.jp
 南部町,www.town.aomori-nanbu.lg.jp
@@ -280,7 +281,7 @@
 大崎市,www.city.osaki.miyagi.jp
 富谷市,www.tomiya-city.miyagi.jp
 蔵王町,www.town.zao.miyagi.jp
-七ケ宿町,town.shichikashuku.miyagi.jp
+七ヶ宿町,www.town.shichikashuku.miyagi.jp
 大河原町,www.town.ogawara.miyagi.jp
 村田町,www.town.murata.miyagi.jp
 柴田町,www.town.shibata.miyagi.jp
@@ -301,7 +302,7 @@
 女川町,www.town.onagawa.miyagi.jp
 南三陸町,www.town.minamisanriku.miyagi.jp
 秋田市,www.city.akita.akita.jp
-能代市,www.city.noshiro.akita.jp
+能代市,www.city.noshiro.lg.jp
 横手市,www.city.yokote.lg.jp
 大館市,www.city.odate.akita.jp
 男鹿市,www.city.oga.akita.jp
@@ -379,7 +380,7 @@
 大玉村,www.vill.otama.fukushima.jp
 鏡石町,www.town.kagamiishi.fukushima.jp
 天栄村,www.vill.tenei.fukushima.jp
-下郷町,www.town.shimogo.fukushima.jp
+下郷町,town.shimogo.fukushima.jp
 檜枝岐村,www.hinoemata.com
 只見町,www.tadami.gr.jp
 南会津町,www.minamiaizu.org
@@ -434,7 +435,7 @@
 笠間市,www.city.kasama.lg.jp
 取手市,www.city.toride.ibaraki.jp
 牛久市,www.city.ushiku.lg.jp
-つくば市,www.city.tsukuba.ibaraki.jp
+つくば市,www.city.tsukuba.lg.jp
 ひたちなか市,www.city.hitachinaka.lg.jp
 鹿嶋市,www.city.kashima.ibaraki.jp
 潮来市,www.city.itako.lg.jp
@@ -470,7 +471,7 @@
 鹿沼市,www.city.kanuma.tochigi.jp
 日光市,www.city.nikko.lg.jp
 小山市,www.city.oyama.tochigi.jp
-真岡市,www.city.moka.tochigi.jp
+真岡市,www.city.moka.lg.jp
 大田原市,www.city.ohtawara.tochigi.jp
 矢板市,www.city.yaita.tochigi.jp
 那須塩原市,www.city.nasushiobara.lg.jp
@@ -490,7 +491,7 @@
 那珂川町,www.town.tochigi-nakagawa.lg.jp
 前橋市,www.city.maebashi.gunma.jp
 高崎市,www.city.takasaki.gunma.jp
-桐生市,www.city.kiryu.lg.jp
+桐生市,www.city.kiryu.gunma.jp
 伊勢崎市,www.city.isesaki.lg.jp
 太田市,www.city.ota.gunma.jp
 沼田市,www.city.numata.gunma.jp
@@ -512,7 +513,7 @@
 嬬恋村,www.vill.tsumagoi.gunma.jp
 草津町,www.town.kusatsu.gunma.jp
 高山村,vill.takayama.gunma.jp
-東吾妻町,www1.town.higashiagatsuma.gunma.jp
+東吾妻町,www.town.higashiagatsuma.gunma.jp
 片品村,www.vill.katashina.gunma.jp
 川場村,www.vill.kawaba.gunma.jp
 昭和村,www.vill.showa.gunma.jp
@@ -596,6 +597,7 @@
 宮代町,www.town.miyashiro.lg.jp
 杉戸町,www.town.sugito.lg.jp
 松伏町,www.town.matsubushi.lg.jp
+千葉市,www.city.chiba.jp
 千葉市,www.city.chiba.jp
 中央区,www.city.chiba.jp
 花見川区,www.city.chiba.jp
@@ -691,6 +693,7 @@
 小金井市,www.city.koganei.lg.jp
 小平市,www.city.kodaira.tokyo.jp
 日野市,www.city.hino.lg.jp
+日野市,www.city.hino.lg.jp
 東村山市,www.city.higashimurayama.tokyo.jp
 国分寺市,www.city.kokubunji.tokyo.jp
 国立市,www.city.kunitachi.tokyo.jp
@@ -713,7 +716,7 @@
 利島村,www.toshimamura.org
 新島村,www.niijima.com
 神津島村,vill.kouzushima.tokyo.jp
-三宅村,www.squashysoftware.com
+三宅村,www.vill.miyake.tokyo.jp
 御蔵島村,www.mikurasima.jp
 八丈町,www.town.hachijo.tokyo.jp
 青ヶ島村,www.vill.aogashima.tokyo.jp
@@ -749,6 +752,8 @@
 緑区,www.city.sagamihara.kanagawa.jp
 中央区,www.city.sagamihara.kanagawa.jp
 南区,www.city.sagamihara.kanagawa.jp
+横須賀市,www.city.yokosuka.kanagawa.jp.c.rb.hp.transer.com
+横須賀市,www.city.yokosuka.kanagawa.jp.e.rb.hp.transer.com
 横須賀市,www.city.yokosuka.kanagawa.jp
 平塚市,www.city.hiratsuka.kanagawa.jp
 鎌倉市,www.city.kamakura.kanagawa.jp
@@ -791,7 +796,7 @@
 長岡市,www.city.nagaoka.niigata.jp
 三条市,www.city.sanjo.niigata.jp
 柏崎市,www.city.kashiwazaki.lg.jp
-新発田市,www.city.shibata.niigata.jp
+新発田市,www.city.shibata.lg.jp
 小千谷市,www.city.ojiya.niigata.jp
 加茂市,www.city.kamo.niigata.jp
 十日町市,www.city.tokamachi.lg.jp
@@ -918,7 +923,7 @@
 川上村,www.vill.kawakami.nagano.jp
 南牧村,www.minamimakimura.jp
 南相木村,www.minamiaiki.jp
-北相木村,vill.kitaaiki.nagano.jp
+北相木村,www.vill.kitaaiki.nagano.jp
 佐久穂町,www.town.sakuho.nagano.jp
 軽井沢町,www.town.karuizawa.lg.jp
 御代田町,www.town.miyota.nagano.jp
@@ -927,7 +932,7 @@
 長和町,town.nagawa.nagano.jp
 下諏訪町,www.town.shimosuwa.lg.jp
 富士見町,www.town.fujimi.lg.jp
-原村,www.vill.hara.lg.jp
+原村,www.vill.hara.nagano.jp
 辰野町,www.town.tatsuno.nagano.jp
 箕輪町,www.town.minowa.lg.jp
 飯島町,www.town.iijima.lg.jp
@@ -937,7 +942,7 @@
 松川町,www.matsukawa-town.jp
 高森町,www.town.nagano-takamori.lg.jp
 阿南町,www.town.anan.nagano.jp
-阿智村,www.vill.achi.lg.jp
+阿智村,www.vill.achi.nagano.jp
 平谷村,www.vill.hiraya.nagano.jp
 根羽村,nebamura.jp
 下條村,www.vill-shimojo.jp
@@ -966,7 +971,7 @@
 小布施町,www.town.obuse.nagano.jp
 高山村,www.vill.takayama.nagano.jp
 山ノ内町,www.town.yamanouchi.nagano.jp
-木島平村,www.vill.kijimadaira.lg.jp
+木島平村,www.kijimadaira.jp
 野沢温泉村,www.vill.nozawaonsen.nagano.jp
 信濃町,www.town.shinano.lg.jp
 小川村,www.vill.ogawa.nagano.jp
@@ -1029,7 +1034,7 @@
 沼津市,www.city.numazu.shizuoka.jp
 熱海市,www.city.atami.lg.jp
 三島市,www.city.mishima.shizuoka.jp
-富士宮市,www.city.fujinomiya.lg.jp
+富士宮市,www.city.fujinomiya.shizuoka.jp
 伊東市,www.city.ito.shizuoka.jp
 島田市,www.city.shimada.shizuoka.jp
 富士市,www.city.fuji.shizuoka.jp
@@ -1259,7 +1264,7 @@
 茨木市,www.city.ibaraki.osaka.jp
 八尾市,www.city.yao.osaka.jp
 泉佐野市,www.city.izumisano.lg.jp
-富田林市,www.city.tondabayashi.lg.jp
+富田林市,www.city.tondabayashi.osaka.jp
 寝屋川市,www.city.neyagawa.osaka.jp
 河内長野市,www.city.kawachinagano.lg.jp
 松原市,www.city.matsubara.lg.jp
@@ -1317,9 +1322,9 @@
 小野市,www.city.ono.hyogo.jp
 三田市,www.city.sanda.lg.jp
 加西市,www.city.kasai.hyogo.jp
-篠山市,www.city.sasayama.lg.jp
+篠山市,www.city.sasayama.hyogo.jp
 養父市,www.city.yabu.hyogo.jp
-丹波市,www.city.tamba.hyogo.jp
+丹波市,www.city.tamba.lg.jp
 南あわじ市,www.city.minamiawaji.hyogo.jp
 朝来市,www.city.asago.hyogo.jp
 淡路市,www.city.awaji.lg.jp
@@ -1327,7 +1332,7 @@
 加東市,www.city.kato.lg.jp
 たつの市,www.city.tatsuno.lg.jp
 猪名川町,www.town.inagawa.lg.jp
-多可町,www.takacho.jp
+多可町,www.town.taka.lg.jp
 稲美町,www.town.hyogo-inami.lg.jp
 播磨町,www.town.harima.lg.jp
 市川町,www.town.ichikawa.lg.jp
@@ -1361,7 +1366,7 @@
 曽爾村,www.vill.soni.nara.jp
 御杖村,www.vill.mitsue.nara.jp
 高取町,www.town.takatori.nara.jp
-明日香村,asukamura.jp
+明日香村,www.asukamura.jp
 上牧町,www.town.kanmaki.nara.jp
 王寺町,www.town.oji.nara.jp
 広陵町,www.town.koryo.nara.jp
@@ -1501,7 +1506,7 @@
 府中町,www.town.fuchu.hiroshima.jp
 海田町,www.town.kaita.lg.jp
 熊野町,www.town.kumano.hiroshima.jp
-坂町,www.town.saka.lg.jp
+坂町,www.town.saka.hiroshima.jp
 安芸太田町,www.akiota.jp
 北広島町,www.town.kitahiroshima.lg.jp
 大崎上島町,www.town.osakikamijima.hiroshima.jp
@@ -1533,7 +1538,7 @@
 吉野川市,www.city.yoshinogawa.lg.jp
 阿波市,www.city.awa.lg.jp
 美馬市,www.city.mima.lg.jp
-三好市,www.city-miyoshi.jp
+三好市,www.miyoshi.i-tokushima.jp
 勝浦町,www.town.katsuura.lg.jp
 上勝町,www.kamikatsu.jp
 佐那河内村,www.vill.sanagochi.lg.jp
@@ -1562,7 +1567,7 @@
 小豆島町,www.town.shodoshima.lg.jp
 三木町,www.town.miki.lg.jp
 直島町,www.town.naoshima.lg.jp
-宇多津町,www.town.utazu.lg.jp
+宇多津町,town.utazu.kagawa.jp
 綾川町,www.town.ayagawa.lg.jp
 琴平町,www.town.kotohira.kagawa.jp
 多度津町,www.town.tadotsu.kagawa.jp
@@ -1578,7 +1583,7 @@
 四国中央市,www.city.shikokuchuo.ehime.jp
 西予市,www.city.seiyo.ehime.jp
 東温市,www.city.toon.ehime.jp
-上島町,www.town.kamijima.lg.jp
+上島町,www.town.kamijima.ehime.jp
 久万高原町,www.kumakogen.jp
 松前町,www.town.masaki.ehime.jp
 砥部町,www.town.tobe.ehime.jp
@@ -1616,7 +1621,7 @@
 越知町,www.town.ochi.kochi.jp
 檮原町,www.town.yusuhara.kochi.jp
 日高村,www.vill.hidaka.kochi.jp
-津野町,town.kochi-tsuno.lg.jp
+津野町,www.town.kochi-tsuno.lg.jp
 四万十町,www.town.shimanto.lg.jp
 大月町,www.town.otsuki.kochi.jp
 三原村,www.vill.mihara.kochi.jp
@@ -1663,7 +1668,7 @@
 朝倉市,www.city.asakura.lg.jp
 みやま市,www.city.miyama.lg.jp
 糸島市,www.city.itoshima.lg.jp
-那珂川町,www.city.nakagawa.lg.jp
+那珂川市,www.city.nakagawa.lg.jp
 宇美町,www.town.umi.lg.jp
 篠栗町,www.town.sasaguri.fukuoka.jp
 志免町,www.town.shime.lg.jp
@@ -1763,12 +1768,12 @@
 大津町,www.town.ozu.kumamoto.jp
 菊陽町,www.town.kikuyo.lg.jp
 南小国町,www.town.minamioguni.kumamoto.jp
-小国町,www.town.kumamoto-oguni.lg.jp
+小国町,www.aso-oguni.com
 産山村,www.ubuyama-v.jp
 高森町,www.town.takamori.kumamoto.jp
 西原村,www.vill.nishihara.kumamoto.jp
 南阿蘇村,www.vill.minamiaso.lg.jp
-御船町,www.town.mifune.kumamoto.jp
+御船町,portal.kumamoto-net.ne.jp
 嘉島町,www.town.kashima.kumamoto.jp
 益城町,www.town.mashiki.lg.jp
 甲佐町,www.town.kosa.kumamoto.jp
@@ -1801,7 +1806,7 @@
 由布市,www.city.yufu.oita.jp
 国東市,www.city.kunisaki.oita.jp
 姫島村,www.himeshima.jp
-日出町,www.town.hiji.lg.jp
+日出町,www.town.hiji.oita.jp
 九重町,www.town.kokonoe.oita.jp
 玖珠町,www.town.kusu.oita.jp
 宮崎市,www.city.miyazaki.miyazaki.jp
@@ -1858,7 +1863,7 @@
 東串良町,www.higashikushira.com
 錦江町,www.town.kinko.lg.jp
 南大隅町,www.town.minamiosumi.lg.jp
-肝付町,kimotsuki-town.jp
+肝付町,www.kimotsuki-town.jp
 中種子町,town.nakatane.kagoshima.jp
 南種子町,www.town.minamitane.kagoshima.jp
 屋久島町,www.town.yakushima.kagoshima.jp
