--- codeforfukui
+++ wikidata
@@ -12,6 +12,7 @@
 函館市,www.city.hakodate.hokkaido.jp
 小樽市,www.city.otaru.lg.jp
 旭川市,www.city.asahikawa.hokkaido.jp
+室蘭市,www.city.muroran.lg.jp
 室蘭市,www.city.muroran.lg.jp
 釧路市,www.city.kushiro.lg.jp
 帯広市,www.city.obihiro.hokkaido.jp
@@ -45,7 +46,7 @@
 北斗市,www.city.hokuto.hokkaido.jp
 当別町,www.town.tobetsu.hokkaido.jp
 新篠津村,www.vill.shinshinotsu.hokkaido.jp
-松前町,www.e-matsumae.com
+松前町,www.town.matsumae.hokkaido.jp
 福島町,www.town.fukushima.hokkaido.jp
 知内町,www.town.shiriuchi.hokkaido.jp
 木古内町,www.town.kikonai.hokkaido.jp
@@ -95,7 +96,7 @@
 北竜町,www.town.hokuryu.hokkaido.jp
 沼田町,www.town.numata.hokkaido.jp
 鷹栖町,www.town.takasu.hokkaido.jp
-東神楽町,town.higashikagura.hokkaido.jp
+東神楽町,www.town.higashikagura.lg.jp
 当麻町,town.tohma.hokkaido.jp
 比布町,www.town.pippu.hokkaido.jp
 愛別町,www.town.aibetsu.hokkaido.jp
@@ -220,10 +221,10 @@
 大間町,www.town.ooma.lg.jp
 東通村,www.vill.higashidoori.lg.jp
 風間浦村,www.kazamaura.jp
-佐井村,www.sai.e-shimokita.jp
-三戸町,town.sannohe.aomori.jp
+佐井村,www.vill.sai.lg.jp
+三戸町,www.town.sannohe.aomori.jp
 五戸町,www.town.gonohe.aomori.jp
-田子町,www.town.takko.aomori.jp
+田子町,www.town.takko.lg.jp
 南部町,www.town.aomori-nanbu.lg.jp
 階上町,www.town.hashikami.lg.jp
 新郷村,www.vill.shingo.aomori.jp
@@ -278,8 +279,9 @@
 栗原市,www.kuriharacity.jp
 東松島市,www.city.higashimatsushima.miyagi.jp
 大崎市,www.city.osaki.miyagi.jp
+富谷市,www.tomiya-city.miyagi.jp
 蔵王町,www.town.zao.miyagi.jp
-七ケ宿町,www.town.shichikashuku.miyagi.jp
+七ヶ宿町,www.town.shichikashuku.miyagi.jp
 大河原町,www.town.ogawara.miyagi.jp
 村田町,www.town.murata.miyagi.jp
 柴田町,www.town.shibata.miyagi.jp
@@ -292,7 +294,6 @@
 利府町,www.town.rifu.miyagi.jp
 大和町,www.town.taiwa.miyagi.jp
 大郷町,www.town.miyagi-osato.lg.jp
-富谷町,www.town.tomiya.miyagi.jp
 大衡村,www.village.ohira.miyagi.jp
 色麻町,www.town.shikama.miyagi.jp
 加美町,www.town.kami.miyagi.jp
@@ -301,7 +302,7 @@
 女川町,www.town.onagawa.miyagi.jp
 南三陸町,www.town.minamisanriku.miyagi.jp
 秋田市,www.city.akita.akita.jp
-能代市,www.city.noshiro.akita.jp
+能代市,www.city.noshiro.lg.jp
 横手市,www.city.yokote.lg.jp
 大館市,www.city.odate.akita.jp
 男鹿市,www.city.oga.akita.jp
@@ -318,7 +319,7 @@
 藤里町,www.town.fujisato.akita.jp
 三種町,www.town.mitane.akita.jp
 八峰町,www.town.happou.akita.jp
-五城目町,www.cs.town.gojome.akita.jp
+五城目町,www.town.gojome.akita.jp
 八郎潟町,www.town.hachirogata.akita.jp
 井川町,www.town.ikawa.akita.jp
 大潟村,www.ogata.or.jp
@@ -430,7 +431,7 @@
 常総市,www.city.joso.lg.jp
 常陸太田市,www.city.hitachiota.ibaraki.jp
 高萩市,www.city.takahagi.ibaraki.jp
-北茨城市,www.city-kitaibaraki.jp
+北茨城市,www.city.kitaibaraki.lg.jp
 笠間市,www.city.kasama.lg.jp
 取手市,www.city.toride.ibaraki.jp
 牛久市,www.city.ushiku.lg.jp
@@ -498,7 +499,7 @@
 渋川市,www.city.shibukawa.lg.jp
 藤岡市,www.city.fujioka.gunma.jp
 富岡市,www.city.tomioka.lg.jp
-安中市,www.city.annaka.gunma.jp
+安中市,www.city.annaka.lg.jp
 みどり市,www.city.midori.gunma.jp
 榛東村,www.vill.shinto.gunma.jp
 吉岡町,www.town.yoshioka.gunma.jp
@@ -512,7 +513,7 @@
 嬬恋村,www.vill.tsumagoi.gunma.jp
 草津町,www.town.kusatsu.gunma.jp
 高山村,vill.takayama.gunma.jp
-東吾妻町,www1.town.higashiagatsuma.gunma.jp
+東吾妻町,www.town.higashiagatsuma.gunma.jp
 片品村,www.vill.katashina.gunma.jp
 川場村,www.vill.kawaba.gunma.jp
 昭和村,www.vill.showa.gunma.jp
@@ -554,7 +555,7 @@
 越谷市,www.city.koshigaya.saitama.jp
 蕨市,www.city.warabi.saitama.jp
 戸田市,www.city.toda.saitama.jp
-入間市,www.city.iruma.lg.jp
+入間市,www.city.iruma.saitama.jp
 朝霞市,www.city.asaka.lg.jp
 志木市,www.city.shiki.lg.jp
 和光市,www.city.wako.lg.jp
@@ -593,9 +594,10 @@
 神川町,www.town.kamikawa.saitama.jp
 上里町,www.town.kamisato.saitama.jp
 寄居町,www.town.yorii.saitama.jp
-宮代町,www.town.miyashiro.saitama.jp
+宮代町,www.town.miyashiro.lg.jp
 杉戸町,www.town.sugito.lg.jp
 松伏町,www.town.matsubushi.lg.jp
+千葉市,www.city.chiba.jp
 千葉市,www.city.chiba.jp
 中央区,www.city.chiba.jp
 花見川区,www.city.chiba.jp
@@ -691,6 +693,7 @@
 小金井市,www.city.koganei.lg.jp
 小平市,www.city.kodaira.tokyo.jp
 日野市,www.city.hino.lg.jp
+日野市,www.city.hino.lg.jp
 東村山市,www.city.higashimurayama.tokyo.jp
 国分寺市,www.city.kokubunji.tokyo.jp
 国立市,www.city.kunitachi.tokyo.jp
@@ -750,6 +753,7 @@
 中央区,www.city.sagamihara.kanagawa.jp
 南区,www.city.sagamihara.kanagawa.jp
 横須賀市,www.city.yokosuka.kanagawa.jp
+横須賀市,www.city.yokosuka.kanagawa.jp
 平塚市,www.city.hiratsuka.kanagawa.jp
 鎌倉市,www.city.kamakura.kanagawa.jp
 藤沢市,www.city.fujisawa.kanagawa.jp
@@ -806,7 +810,7 @@
 佐渡市,www.city.sado.niigata.jp
 魚沼市,www.city.uonuma.niigata.jp
 南魚沼市,www.city.minamiuonuma.niigata.jp
-胎内市,www.city.tainai.lg.jp
+胎内市,www.city.tainai.niigata.jp
 聖籠町,www.town.seiro.niigata.jp
 弥彦村,www.vill.yahiko.niigata.jp
 田上町,www.town.tagami.niigata.jp
@@ -909,7 +913,7 @@
 大町市,www.city.omachi.nagano.jp
 飯山市,www.city.iiyama.nagano.jp
 茅野市,www.city.chino.lg.jp
-塩尻市,www.city.shiojiri.nagano.jp
+塩尻市,www.city.shiojiri.lg.jp
 佐久市,www.city.saku.nagano.jp
 千曲市,www.city.chikuma.lg.jp
 東御市,www.city.tomi.nagano.jp
@@ -968,7 +972,7 @@
 山ノ内町,www.town.yamanouchi.nagano.jp
 木島平村,www.kijimadaira.jp
 野沢温泉村,www.vill.nozawaonsen.nagano.jp
-信濃町,www.town.shinanomachi.nagano.jp
+信濃町,www.town.shinano.lg.jp
 小川村,www.vill.ogawa.nagano.jp
 飯綱町,www.town.iizuna.nagano.jp
 栄村,www.vill.sakae.nagano.jp
@@ -1368,7 +1372,7 @@
 河合町,www.town.kawai.nara.jp
 吉野町,www.town.yoshino.nara.jp
 大淀町,www.town.oyodo.lg.jp
-下市町,www.town.shimoichi.nara.jp
+下市町,www.town.shimoichi.lg.jp
 黒滝村,www.vill.kurotaki.nara.jp
 天川村,www.vill.tenkawa.nara.jp
 野迫川村,www.vill.nosegawa.nara.jp
@@ -1443,7 +1447,7 @@
 吉賀町,www.town.yoshika.lg.jp
 海士町,www.town.ama.shimane.jp
 西ノ島町,www.town.nishinoshima.shimane.jp
-知夫村,chibu-vill.com
+知夫村,www.vill.chibu.lg.jp
 隠岐の島町,www.town.okinoshima.shimane.jp
 岡山市,www.city.okayama.jp
 北区,www.city.okayama.jp
@@ -1533,7 +1537,7 @@
 吉野川市,www.city.yoshinogawa.lg.jp
 阿波市,www.city.awa.lg.jp
 美馬市,www.city.mima.lg.jp
-三好市,www.city-miyoshi.jp
+三好市,www.miyoshi.i-tokushima.jp
 勝浦町,www.town.katsuura.lg.jp
 上勝町,www.kamikatsu.jp
 佐那河内村,www.vill.sanagochi.lg.jp
@@ -1579,7 +1583,7 @@
 西予市,www.city.seiyo.ehime.jp
 東温市,www.city.toon.ehime.jp
 上島町,www.town.kamijima.ehime.jp
-久万高原町,www.town.kumakogen.ehime.jp
+久万高原町,www.kumakogen.jp
 松前町,www.town.masaki.ehime.jp
 砥部町,www.town.tobe.ehime.jp
 内子町,www.town.uchiko.ehime.jp
@@ -1673,7 +1677,7 @@
 粕屋町,www.town.kasuya.fukuoka.jp
 芦屋町,www.town.ashiya.lg.jp
 水巻町,www.town.mizumaki.lg.jp
-岡垣町,www.town.okagaki.fukuoka.jp
+岡垣町,www.town.okagaki.lg.jp
 遠賀町,www.town.onga.lg.jp
 小竹町,town.kotake.lg.jp
 鞍手町,www.town.kurate.lg.jp
@@ -1775,7 +1779,7 @@
 山都町,www.town.kumamoto-yamato.lg.jp
 氷川町,www.hikawacyou.hinokuni-net.jp
 芦北町,www.ashikita-t.kumamoto-sgn.jp
-津奈木町,portal.kumamoto-net.ne.jp
+津奈木町,www.town.tsunagi.lg.jp
 錦町,www.nishiki-machi.com
 多良木町,www.town.taragi.lg.jp
 湯前町,www.town.yunomae.lg.jp
@@ -1811,7 +1815,7 @@
 小林市,www.city.kobayashi.lg.jp
 日向市,www.city.hyuga.miyazaki.jp
 串間市,www.city.kushima.lg.jp
-西都市,www.city.saito.miyazaki.jp
+西都市,www.saito-city.jp
 えびの市,www.city.ebino.lg.jp
 三股町,www.town.mimata.lg.jp
 高原町,www.town.takaharu.lg.jp
@@ -1819,20 +1823,20 @@
 綾町,www.town.aya.miyazaki.jp
 高鍋町,www.town.takanabe.miyazaki.jp
 新富町,www.town.shintomi.miyazaki.jp
-西米良村,www.nishimera.jp
+西米良村,www.vill.nishimera.lg.jp
 木城町,www.kijo.jp
 川南町,www.town.kawaminami.miyazaki.jp
-都農町,www.town.tsuno.miyazaki.jp
+都農町,www.town.tsuno.lg.jp
 門川町,www.town.kadogawa.lg.jp
 諸塚村,www.vill.morotsuka.miyazaki.jp
 椎葉村,www.vill.shiiba.miyazaki.jp
 美郷町,www.town.miyazaki-misato.lg.jp
 高千穂町,www.town-takachiho.jp
-日之影町,www.hinokage.jp
+日之影町,www.town.hinokage.lg.jp
 五ヶ瀬町,www.town.gokase.miyazaki.jp
 鹿児島市,www.city.kagoshima.lg.jp
 鹿屋市,www.e-kanoya.net
-枕崎市,www.city.makurazaki.kagoshima.jp
+枕崎市,www.city.makurazaki.lg.jp
 阿久根市,www.city.akune.kagoshima.jp
 出水市,www.city.izumi.kagoshima.jp
 指宿市,www.city.ibusuki.lg.jp
@@ -1864,7 +1868,7 @@
 屋久島町,www.town.yakushima.kagoshima.jp
 大和村,www.vill.yamato.lg.jp
 宇検村,www.uken.net
-瀬戸内町,www.bookkyotohotels.com
+瀬戸内町,www.town.setouchi.lg.jp
 龍郷町,www.town.tatsugo.lg.jp
 喜界町,www.town.kikai.lg.jp
 徳之島町,www.tokunoshima-town.org
