--- codeforfukui
+++ wikidata
@@ -31,7 +31,7 @@
 名寄市,www.city.nayoro.lg.jp
 三笠市,www.city.mikasa.hokkaido.jp
 根室市,www.city.nemuro.hokkaido.jp
-千歳市,www.city.chitose.hokkaido.jp
+千歳市,www.city.chitose.lg.jp
 滝川市,www.city.takikawa.hokkaido.jp
 砂川市,www.city.sunagawa.hokkaido.jp
 歌志内市,www.city.utashinai.hokkaido.jp
@@ -66,7 +66,7 @@
 黒松内町,www.kuromatsunai.com
 蘭越町,www.town.rankoshi.hokkaido.jp
 ニセコ町,www.town.niseko.lg.jp
-真狩村,makkari.info
+真狩村,www.vill.makkari.lg.jp
 留寿都村,www.vill.rusutsu.lg.jp
 喜茂別町,www.town.kimobetsu.hokkaido.jp
 京極町,www.town-kyogoku.jp
@@ -123,7 +123,7 @@
 猿払村,www.vill.sarufutsu.hokkaido.jp
 浜頓別町,www.town.hamatonbetsu.hokkaido.jp
 中頓別町,www.town.nakatombetsu.hokkaido.jp
-枝幸町,www.town.esashi.hokkaido.jp
+枝幸町,www.esashi.jp
 豊富町,www.town.toyotomi.hokkaido.jp
 礼文町,www.town.rebun.hokkaido.jp
 利尻町,www.town.rishiri.hokkaido.jp
@@ -221,7 +221,7 @@
 東通村,www.vill.higashidoori.lg.jp
 風間浦村,www.kazamaura.jp
 佐井村,www.vill.sai.lg.jp
-三戸町,town.sannohe.aomori.jp
+三戸町,www.town.sannohe.aomori.jp
 五戸町,www.town.gonohe.aomori.jp
 田子町,www.town.takko.lg.jp
 南部町,www.town.aomori-nanbu.lg.jp
@@ -280,7 +280,7 @@
 大崎市,www.city.osaki.miyagi.jp
 富谷市,www.tomiya-city.miyagi.jp
 蔵王町,www.town.zao.miyagi.jp
-七ケ宿町,town.shichikashuku.miyagi.jp
+七ヶ宿町,town.shichikashuku.miyagi.jp
 大河原町,www.town.ogawara.miyagi.jp
 村田町,www.town.murata.miyagi.jp
 柴田町,www.town.shibata.miyagi.jp
@@ -362,7 +362,7 @@
 遊佐町,www.town.yuza.yamagata.jp
 福島市,www.city.fukushima.fukushima.jp
 会津若松市,www.city.aizuwakamatsu.fukushima.jp
-郡山市,www.city.koriyama.fukushima.jp
+郡山市,www.city.koriyama.lg.jp
 いわき市,www.city.iwaki.lg.jp
 白河市,www.city.shirakawa.fukushima.jp
 須賀川市,www.city.sukagawa.fukushima.jp
@@ -381,7 +381,7 @@
 天栄村,www.vill.tenei.fukushima.jp
 下郷町,www.town.shimogo.fukushima.jp
 檜枝岐村,www.hinoemata.com
-只見町,www.tadami.gr.jp
+只見町,www.town.tadami.lg.jp
 南会津町,www.minamiaizu.org
 北塩原村,www.vill.kitashiobara.fukushima.jp
 西会津町,www.town.nishiaizu.fukushima.jp
@@ -434,7 +434,7 @@
 笠間市,www.city.kasama.lg.jp
 取手市,www.city.toride.ibaraki.jp
 牛久市,www.city.ushiku.lg.jp
-つくば市,www.city.tsukuba.ibaraki.jp
+つくば市,www.city.tsukuba.lg.jp
 ひたちなか市,www.city.hitachinaka.lg.jp
 鹿嶋市,www.city.kashima.ibaraki.jp
 潮来市,www.city.itako.lg.jp
@@ -470,18 +470,18 @@
 鹿沼市,www.city.kanuma.tochigi.jp
 日光市,www.city.nikko.lg.jp
 小山市,www.city.oyama.tochigi.jp
-真岡市,www.city.moka.tochigi.jp
+真岡市,www.city.moka.lg.jp
 大田原市,www.city.ohtawara.tochigi.jp
 矢板市,www.city.yaita.tochigi.jp
 那須塩原市,www.city.nasushiobara.lg.jp
 さくら市,www.city.tochigi-sakura.lg.jp
 那須烏山市,www.city.nasukarasuyama.lg.jp
 下野市,www.city.shimotsuke.lg.jp
-上三川町,www.town.kaminokawa.tochigi.jp
+上三川町,www.town.kaminokawa.lg.jp
 益子町,www.town.mashiko.tochigi.jp
 茂木町,www.town.motegi.tochigi.jp
 市貝町,www.town.ichikai.tochigi.jp
-芳賀町,www.town.haga.tochigi.jp
+芳賀町,www.town.tochigi-haga.lg.jp
 壬生町,www.town.mibu.tochigi.jp
 野木町,www.town.nogi.lg.jp
 塩谷町,www.town.shioya.tochigi.jp
@@ -502,17 +502,17 @@
 みどり市,www.city.midori.gunma.jp
 榛東村,www.vill.shinto.gunma.jp
 吉岡町,www.town.yoshioka.gunma.jp
-上野村,www.uenomura.ne.jp
+上野村,www.uenomura.jp
 神流町,www.town.kanna.gunma.jp
 下仁田町,www.town.shimonita.lg.jp
 南牧村,www.nanmoku.ne.jp
 甘楽町,www.town.kanra.gunma.jp
-中之条町,www.town.nakanojo.gunma.jp
+中之条町,town.nakanojo.gunma.jp
 長野原町,www.town.naganohara.gunma.jp
 嬬恋村,www.vill.tsumagoi.gunma.jp
 草津町,www.town.kusatsu.gunma.jp
 高山村,vill.takayama.gunma.jp
-東吾妻町,www1.town.higashiagatsuma.gunma.jp
+東吾妻町,www.town.higashiagatsuma.gunma.jp
 片品村,www.vill.katashina.gunma.jp
 川場村,www.vill.kawaba.gunma.jp
 昭和村,www.vill.showa.gunma.jp
@@ -713,7 +713,7 @@
 利島村,www.toshimamura.org
 新島村,www.niijima.com
 神津島村,vill.kouzushima.tokyo.jp
-三宅村,www.squashysoftware.com
+三宅村,www.vill.miyake.tokyo.jp
 御蔵島村,www.mikurasima.jp
 八丈町,www.town.hachijo.tokyo.jp
 青ヶ島村,www.vill.aogashima.tokyo.jp
@@ -749,6 +749,8 @@
 緑区,www.city.sagamihara.kanagawa.jp
 中央区,www.city.sagamihara.kanagawa.jp
 南区,www.city.sagamihara.kanagawa.jp
+横須賀市,www.city.hakodate.hokkaido.jp
+横須賀市,www.city.hakodate.hokkaido.jp
 横須賀市,www.city.yokosuka.kanagawa.jp
 平塚市,www.city.hiratsuka.kanagawa.jp
 鎌倉市,www.city.kamakura.kanagawa.jp
@@ -791,7 +793,7 @@
 長岡市,www.city.nagaoka.niigata.jp
 三条市,www.city.sanjo.niigata.jp
 柏崎市,www.city.kashiwazaki.lg.jp
-新発田市,www.city.shibata.niigata.jp
+新発田市,www.city.shibata.lg.jp
 小千谷市,www.city.ojiya.niigata.jp
 加茂市,www.city.kamo.niigata.jp
 十日町市,www.city.tokamachi.lg.jp
@@ -822,7 +824,7 @@
 魚津市,www.city.uozu.toyama.jp
 氷見市,www.city.himi.toyama.jp
 滑川市,www.city.namerikawa.toyama.jp
-黒部市,kurobe.city.kurobe.toyama.jp
+黒部市,www.city.kurobe.toyama.jp
 砺波市,www.city.tonami.toyama.jp
 小矢部市,www.city.oyabe.toyama.jp
 南砺市,www.city.nanto.toyama.jp
@@ -836,7 +838,7 @@
 七尾市,www.city.nanao.lg.jp
 小松市,www.city.komatsu.lg.jp
 輪島市,www.city.wajima.ishikawa.jp
-珠洲市,www.city.suzu.ishikawa.jp
+珠洲市,www.city.suzu.lg.jp
 加賀市,www.city.kaga.ishikawa.jp
 羽咋市,www.city.hakui.lg.jp
 かほく市,www.city.kahoku.ishikawa.jp
@@ -931,14 +933,14 @@
 辰野町,www.town.tatsuno.nagano.jp
 箕輪町,www.town.minowa.lg.jp
 飯島町,www.town.iijima.lg.jp
-南箕輪村,www.vill.minamiminowa.nagano.jp
+南箕輪村,www.vill.minamiminowa.lg.jp
 中川村,www.vill.nakagawa.nagano.jp
 宮田村,www.vill.miyada.nagano.jp
-松川町,www.matsukawa-town.jp
+松川町,www.town.matsukawa.lg.jp
 高森町,www.town.nagano-takamori.lg.jp
 阿南町,www.town.anan.nagano.jp
 阿智村,www.vill.achi.lg.jp
-平谷村,www.vill.hiraya.nagano.jp
+平谷村,hirayamura.jp
 根羽村,nebamura.jp
 下條村,www.vill-shimojo.jp
 売木村,www.urugi.jp
@@ -1054,7 +1056,7 @@
 西伊豆町,www.town.nishiizu.shizuoka.jp
 函南町,www.town.kannami.shizuoka.jp
 清水町,www.town.shimizu.shizuoka.jp
-長泉町,japan.nagaizumi.org
+長泉町,www.town.nagaizumi.lg.jp
 小山町,www.fuji-oyama.jp
 吉田町,www.town.yoshida.shizuoka.jp
 川根本町,www.town.kawanehon.shizuoka.jp
@@ -1084,7 +1086,7 @@
 春日井市,www.city.kasugai.lg.jp
 豊川市,www.city.toyokawa.lg.jp
 津島市,www.city.tsushima.lg.jp
-碧南市,www.city.hekinan.aichi.jp
+碧南市,www.city.hekinan.lg.jp
 刈谷市,www.city.kariya.lg.jp
 豊田市,www.city.toyota.aichi.jp
 安城市,www.city.anjo.aichi.jp
@@ -1115,7 +1117,7 @@
 長久手市,www.city.nagakute.lg.jp
 東郷町,www.town.aichi-togo.lg.jp
 豊山町,www.town.toyoyama.lg.jp
-大口町,www.town.oguchi.aichi.jp
+大口町,www.town.oguchi.lg.jp
 扶桑町,www.town.fuso.lg.jp
 大治町,www.town.oharu.aichi.jp
 蟹江町,www.town.kanie.aichi.jp
@@ -1189,7 +1191,7 @@
 伏見区,www.city.kyoto.lg.jp
 山科区,www.city.kyoto.lg.jp
 西京区,www.city.kyoto.lg.jp
-福知山市,www.city.fukuchiyama.kyoto.jp
+福知山市,www.city.fukuchiyama.lg.jp
 舞鶴市,www.city.maizuru.kyoto.jp
 綾部市,www.city.ayabe.lg.jp
 宇治市,www.city.uji.kyoto.jp
@@ -1312,14 +1314,14 @@
 西脇市,www.city.nishiwaki.lg.jp
 宝塚市,www.city.takarazuka.hyogo.jp
 三木市,www.city.miki.lg.jp
-高砂市,www.city.takasago.hyogo.jp
+高砂市,www.city.takasago.lg.jp
 川西市,www.city.kawanishi.hyogo.jp
 小野市,www.city.ono.hyogo.jp
 三田市,www.city.sanda.lg.jp
 加西市,www.city.kasai.hyogo.jp
-篠山市,www.city.sasayama.lg.jp
+篠山市,www.city.sasayama.hyogo.jp
 養父市,www.city.yabu.hyogo.jp
-丹波市,www.city.tamba.hyogo.jp
+丹波市,www.city.tamba.lg.jp
 南あわじ市,www.city.minamiawaji.hyogo.jp
 朝来市,www.city.asago.hyogo.jp
 淡路市,www.city.awaji.lg.jp
@@ -1327,7 +1329,7 @@
 加東市,www.city.kato.lg.jp
 たつの市,www.city.tatsuno.lg.jp
 猪名川町,www.town.inagawa.lg.jp
-多可町,www.takacho.jp
+多可町,www.town.taka.lg.jp
 稲美町,www.town.hyogo-inami.lg.jp
 播磨町,www.town.harima.lg.jp
 市川町,www.town.ichikawa.lg.jp
@@ -1464,7 +1466,7 @@
 真庭市,www.city.maniwa.lg.jp
 美作市,www.city.mimasaka.lg.jp
 浅口市,www.city.asakuchi.lg.jp
-和気町,www.town.wake.okayama.jp
+和気町,www.town.wake.lg.jp
 早島町,www.town.hayashima.lg.jp
 里庄町,www.town.satosho.okayama.jp
 矢掛町,www.town.yakage.lg.jp
@@ -1494,7 +1496,7 @@
 三次市,www.city.miyoshi.hiroshima.jp
 庄原市,www.city.shobara.hiroshima.jp
 大竹市,www.city.otake.hiroshima.jp
-東広島市,www.city.higashihiroshima.hiroshima.jp
+東広島市,www.city.higashihiroshima.lg.jp
 廿日市市,www.city.hatsukaichi.hiroshima.jp
 安芸高田市,www.akitakata.jp
 江田島市,www.city.etajima.hiroshima.jp
@@ -1533,7 +1535,7 @@
 吉野川市,www.city.yoshinogawa.lg.jp
 阿波市,www.city.awa.lg.jp
 美馬市,www.city.mima.lg.jp
-三好市,www.city-miyoshi.jp
+三好市,www.miyoshi.i-tokushima.jp
 勝浦町,www.town.katsuura.lg.jp
 上勝町,www.kamikatsu.jp
 佐那河内村,www.vill.sanagochi.lg.jp
@@ -1663,7 +1665,7 @@
 朝倉市,www.city.asakura.lg.jp
 みやま市,www.city.miyama.lg.jp
 糸島市,www.city.itoshima.lg.jp
-那珂川町,www.city.nakagawa.lg.jp
+那珂川市,www.city.nakagawa.lg.jp
 宇美町,www.town.umi.lg.jp
 篠栗町,www.town.sasaguri.fukuoka.jp
 志免町,www.town.shime.lg.jp
@@ -1679,7 +1681,7 @@
 鞍手町,www.town.kurate.lg.jp
 桂川町,www.town.keisen.fukuoka.jp
 筑前町,www.town.chikuzen.fukuoka.jp
-東峰村,www1.vill.toho.fukuoka.jp
+東峰村,vill.toho-info.com
 大刀洗町,www.town.tachiarai.fukuoka.jp
 大木町,www.town.ooki.lg.jp
 広川町,www.town.hirokawa.fukuoka.jp
@@ -1818,7 +1820,7 @@
 国富町,www.town.kunitomi.miyazaki.jp
 綾町,www.town.aya.miyazaki.jp
 高鍋町,www.town.takanabe.miyazaki.jp
-新富町,www.town.shintomi.miyazaki.jp
+新富町,www.town.shintomi.lg.jp
 西米良村,www.vill.nishimera.lg.jp
 木城町,www.kijo.jp
 川南町,www.town.kawaminami.miyazaki.jp
@@ -1868,7 +1870,7 @@
 龍郷町,www.town.tatsugo.lg.jp
 喜界町,www.town.kikai.lg.jp
 徳之島町,www.tokunoshima-town.org
-天城町,www.yui-amagi.com
+天城町,www.town.amagi.lg.jp
 伊仙町,www.town.isen.kagoshima.jp
 和泊町,www.town.wadomari.lg.jp
 知名町,www.town.china.lg.jp
@@ -1891,7 +1893,7 @@
 本部町,www.town.motobu.okinawa.jp
 恩納村,www.vill.onna.okinawa.jp
 宜野座村,www.vill.ginoza.okinawa.jp
-金武町,www.town.kin.okinawa.jp
+金武町,www.city.naha.okinawa.jp
 伊江村,www.iejima.org
 読谷村,www.yomitan.jp
 嘉手納町,www.town.kadena.okinawa.jp
@@ -1910,7 +1912,7 @@
 伊平屋村,www.vill.iheya.okinawa.jp
 伊是名村,vill.izena.okinawa.jp
 久米島町,www.town.kumejima.okinawa.jp
-八重瀬町,www.town.yaese.okinawa.jp
+八重瀬町,www.town.yaese.lg.jp
 多良間村,www.vill.tarama.okinawa.jp
 竹富町,www.town.taketomi.lg.jp
 与那国町,www.town.yonaguni.okinawa.jp
